website/static/files/build_orca_inventory_refined.sql

```diff
@@ -0,0 +1,140 @@
+-- Set the Role and Path
+SET ROLE orca_dbo;
+SET search_path TO orca, public;
+
+
+-- Create the objects in a transaction.
+BEGIN
+
+    -- Create the Providers Table
+    CREATE TABLE IF NOT EXISTS providers
+    (
+      provider_id text NOT NULL
+    , name        text NOT NULL
+    , CONSTRAINT PK_providers PRIMARY KEY(provider_id)
+    );
+
+    COMMENT ON TABLE providers
+      IS 'Providers that are in the ORCA holdings.';
+    COMMENT ON COLUMN providers.provider_id
+      IS 'Provider ID supplied by Cumulus';
+    COMMENT ON COLUMN providers.name
+      IS 'User friendly name of the provider provided by Cumulus';
+
+
+    -- Create the Collections Table
+    CREATE TABLE IF NOT EXISTS collections
+    (
+      collection_id   text NOT NULL
+    , shortname       text NOT NULL
+    , version         text NOT NULL
+    , CONSTRAINT PK_collections PRIMARY KEY(collection_id)
+    , CONSTRAINT UNIQUE_collections UNIQUE (shortname, version)
+    );
+
+    COMMENT ON TABLE collections
+      IS 'Collections that are in the ORCA archive holdings.';
+    COMMENT ON COLUMN collections.collection_id
+      IS 'Collection ID from Cumulus usually in the format shortname__version.';
+    COMMENT ON COLUMN collections.shortname
+      IS 'Collection short name from Cumulus';
+    COMMENT ON COLUMN collections.version
+      IS 'Collection version from Cumulus';
+
+    -- Create the Providers Collection XREF table
+    CREATE TABLE IF NOT EXISTS provider_collection_xref
+    (
+      provider_id     text NOT NULL
+    , collection_id   text NOT NULL
+    , CONSTRAINT PK_provider_collection_xref PRIMARY KEY(provider_id,collection_id)
+    , CONSTRAINT provider_collection_fk FOREIGN KEY(provider_id) REFERENCES providers(provider_id)
+    , CONSTRAINT collection_provider_fk FOREIGN KEY(collection_id) REFERENCES collections(collection_id)
+    );
+
+    COMMENT ON TABLE provider_collection_xref
+      IS 'Cross refrence table that ties a collection and provider together and resolves the many to many relationship.';
+    COMMENT ON COLUMN provider_collection_xref.provider_id
+      IS 'Provider ID from the providers table.';
+    COMMENT ON COLUMN provider_collection_xref.collection_id
+      IS 'Collection ID from the collections table.';
+
+
+    -- Create the Granules table
+    CREATE TABLE IF NOT EXISTS granules
+    (
+      id                  int8 NOT NULL
```

Comment:
`int8` for `id` here should be `bigserial` to match what is in the db_deploy orca_sql.py

Reply:
Side branch